app.py

```diff
@@ -132,9 +132,8 @@
     container_port=4200,
     container_memory=200,
     container_env_vars={
-        "API_DOCS_URL": f"http://{fully_qualified_domain_name}/api-docs",
```

Comment:
sorry, the previous question was meant for this VAR.  Will removing this break the agora app? Does it need a API_DOCS_URL env var to be defined?

Reply:
The web app will not need this to be defined.